### ISSUE TITLE
Add streets table from data.gov

### ISSUE THREAD

**Issue by atalyaalon** [data-for-change/anyway#2478]
The list in data.gov [is here](https://data.gov.il/dataset/321)
1. Add it to our DB
2. Create a flow in Airflow that updates it once a week
3. Decide where to use it and how in our streets cache, discuss with @tkalir and me, and let's implement

**Comment by ziv17** [comment]
Hi @atalyaalon,

- this is still relevant, right?
- do you mean to download manually, or automatically as part of the insert? 
- If automatically, is there an endpoint for API, or do I need to parse the HTML?

**Comment by atalyaalon** [comment]
@ziv17 I suggest you take this next task , since this is relevant for Tamar's work
(I know there is Streets table, not sure when and where it's used, but you can find this mistery and perhaps use this table for this load)

I suggest 
- storing it directly in our DB (and no need to save the file itself in S3)
- use this file: https://data.gov.il/dataset/321/resource/9ad3862c-8391-4b2f-84a4-2d4c68625f4b - there is a button for "API לנתונים"
- Make sure it runs in Airflow once a month
- Regarding Encoding - how do we handle it today in our DB when loading to it hebrew strings from other flows?

**Comment by atalyaalon** [comment]
Following [this issue](https://github.com/data-for-change/anyway/issues/2736) 
I think we need to add to the flow the removal of chars like space, tab, etc
Ziv, What do you think?

**Comment by ziv17** [comment]
The added tab at the end of the street is not in the `streets` table, it is in the news_flash.
Do we have issues of added white space, or erroneous names in the streets table?

**Comment by atalyaalon** [comment]
> The added tab at the end of the street is not in the `streets` table, it is in the news_flash.

Got it. 
I guess we see that since the CBS location added to the newsflash table comes from the CBS data and perhaps they had the tab there..
The hebrew names fo the streets of CBS data in parsers/cbs/executor.py are filled from the CBS dictionary. 

Perhaps we should either: 
- fill the CBS streets from the gov.il streets table
- Wait for [task 2 here](https://github.com/data-for-change/anyway-newsflash-infographics/issues/1177) to be completed. That way querying will be only based on numeric fields and not text fields.

What do you think?

> Do we have issues of added white space, or erroneous names in the streets table?

Not that I know of

**Comment by ziv17** [comment]
Hi @atalyaalon ,
I think `streets` table is already filled from gov.il. I think it was entered in [PR-2668](https://github.com/data-for-change/anyway/pull/2668). I think this issue (#2668 ) can be closed.

**Comment by ziv17** [comment]
>> I guess we see that since the CBS location added to the newsflash table comes from the CBS data and perhaps they had the tab there..

by "CBS data" you mean `cbs_locations` table?

**Comment by atalyaalon** [comment]
> > > I guess we see that since the CBS location added to the newsflash table comes from the CBS data and perhaps they had the tab there..
> 
> by "CBS data" you mean `cbs_locations` table? The street that had the extra `\t` in the news_flash is correct there. Another point. I think I added a change to the news_flash creation that sets also yishuv_symbol and street numeric values. The news_flash from the issue does not have these numeric values.

I mean both `cbs_locations` table and our own CBS tables (markers, markers_hebrew, etc) 
In addition, I checked, and the `\t` appears in cbs_locations - download csv [from here](https://redash.dataforchange.org.il/queries/167/source and open with excel - the tab is there).
For some reason Redash UI does not display this tab but it exist. However we will take care of this in another issue. Great work!

Regarding your second point, you are correct, the street and yishuv_symbol numeric values are still not saved in the newsflashes [see here for example](https://redash.dataforchange.org.il/queries/166/source). Road segment id btw is saved properly. I opened [another issue for it](https://github.com/data-for-change/anyway/issues/2751) and we will take care of it in the coming future.
